libcob/fileio.c

```diff
@@ -211,7 +211,7 @@ DIR					*listdir_handle;
 struct dirent		*listdir_filedata;
 #endif
 
-#define        OPENMODESIZE  3
+#define        OPENMODESIZE  6
```

Comment:
Even if the size of buff is 5, the warning messages will be solved.
Set the size of buff to 5.